manage.py

```
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+"""Django's command-line utility for administrative tasks."""
+import os
+import sys
+
+
+def main():
```

Ruff (0.12.2)

Missing return type annotation for public function `main`

Add return type annotation: `None`

(ANN201)